src/tests/test_pyodide.py

```diff
@@ -1425,3 +1425,26 @@ def test_csp(selenium_standalone_noload):
         selenium.load_pyodide()
     finally:
         target_path.unlink()
+
+
+@pytest.mark.xfail_browsers(node="Browser only")
+def test_static_import(selenium_standalone_noload):
+    selenium = selenium_standalone_noload
+    target_path = DIST_PATH / "module_static_import_test.html"
+    try:
+        shutil.copy(
+            get_pyodide_root() / "src/templates/module_static_import_test.html",
+            target_path,
+        )
+        selenium.goto(f"{selenium.base_url}/module_static_import_test.html")
+        selenium.javascript_setup()
+        selenium.load_pyodide()
+        selenium.run_js(
+            """
+            pyodide.runPython(`
```

Comment:
This test doesn't fail failed prior to the change to `pyodide.ts`. We need to make sure that `${indexURL}/pyodide.asm.js` doesn't exist. Use `tmpdir`, copy the `dist` directory to the temporary directory but remove `pyodide.asm.js` (or move it to a subfolder) and then the static import can be from the correct url. This should fail prior to the change because they dynamic import is from the wrong path.